### DISCUSSION TITLE
Does whatsmeow not yet support registration function?

### DISCUSSION THREAD

**Discussion by ys1231** [tulir/whatsmeow#581] [General]
I cloned whatsmeow and tried running mdtest and found only qr

**Comment by ghost** [comment]
Actually it does, you just need to put the command line while it scan for input below the QR. 
It just doesn't show any prompt like "please input the command : "

![image](https://github.com/tulir/whatsmeow/assets/137609917/e77ae658-2225-4112-b012-25f1c01af511)

**Comment by ys1231** [comment]
I see, thank you very much!
